### ISSUE TITLE
"Download" action on manga update notifications

### ISSUE THREAD

**Issue by pempek** [tachiyomiorg/tachiyomi#2986]
### Why/User Benefit/User Problem
When I receive new chapter notification, I don't always want to read it immediately. But it'd be nice to be able to download the chapter from the notification so when I do want to read the chapter, it's already available and I don't have to download it then.

### What/Requirements
Here's a simple sketch that I made:
![image](https://user-images.githubusercontent.com/36126710/80577363-f6afc880-8a30-11ea-8437-c9415828722b.png)

Basically it adds "Download" quick action to new chapter notification.
When clicked, it automatically downloads the chapter in the background.

**Comment by specterflare** [comment]
Can't you just turn on "Download new chapters" in settings?

**Comment by pempek** [comment]
> Can't you just turn on "Download new chapters" in settings?

I follow quite a lot of mangas. Some of them I'd like to read soon, some of them I'd like to wait for a few chapters before I read. 

I think this is a quick way to prioritize which to download right from the notifications.

**Comment by serjflint** [comment]
Why not download all of them? You can read all of them faster, because they are on the device already. With auto-delete you won't occupy too much space.

**Comment by pempek** [comment]
I don't know if there's anyone else like me, but I follow probably more than 100 mangas and comics. Some of them are mangas I've never read before (might have gone on for many chapters) and I add them as bookmarks only.

Out of those, I only read probably 20 titles regularly. The rest probably once every few weeks. I tend to download only chapters that I plan to read within the day.

I do have another idea. I place my regularly read mangas in a separate category. If I can only download new chapters for that category, it can also work for me.

But I thought that this one is much easier to implement.

**Comment by serjflint** [comment]
> I place my regularly read manga in a separate category. If I can only download new chapters for that category, it can also work for me.

That is that I actually do. There is already an option for an automatic download of certain categories. I have over 1k manga but most of them are planned or already completed.

**Comment by pempek** [comment]
Wait, really? That's awesome. How do you do that? :D

I don't see it on "Downloads" settings.

**Comment by serjflint** [comment]
It is collapsible. You need to enable download new chapters first.

**Comment by serjflint** [comment]
There is also such an option for checking updates. I will die of old age if it will try check all my manga in the library.

**Comment by pempek** [comment]
That's why!
Ok this also works for me. Thank you so much for the information.

**Comment by pempek** [comment]
I think the menu shouldn't be collapsed?

Being greyed out should be enough IMO.

**Comment by Eragonnogare** [comment]
This has sat around for a while, and while the original user found a workaround I would still absolutely love to see this implemented. I wish for this feature probably multiple times every day lol, it would be a godsend. I don't want to reorganize all 300+ manga of mine simply to set up automatic downloads as it would make other things less convenient.
